ext/datadog_cov/datadog_cov.c

```diff
@@ -88,9 +95,25 @@ static VALUE dd_cov_initialize(int argc, VALUE *argv, VALUE self)
   }
   VALUE rb_ignored_path = rb_hash_lookup(opt, ID2SYM(rb_intern("ignored_path")));
 
+  VALUE rb_threading_mode = rb_hash_lookup(opt, ID2SYM(rb_intern("threading_mode")));
+  int threading_mode;
+  if (!RTEST(rb_threading_mode) || rb_threading_mode == ID2SYM(rb_intern("multi")))
+  {
+    threading_mode = MULTI_THREADED_COVERAGE_MODE;
+  }
```

Comment:
Really minor: Consider maybe removing the `RTEST` and allowing it to fall down into the exception. My thinking is, it's kinda weird for `threading_mode: false` to result in the threading mode being activated... :thinking: 

```suggestion
  if (rb_threading_mode == ID2SYM(rb_intern("multi")))
  {
    threading_mode = MULTI_THREADED_COVERAGE_MODE;
  }
```